Application/ResearchDataManagementPlatform/WindowManagement/ActivateItems.cs

```diff
@@ -856,9 +861,27 @@ public override IPipelineRunner GetPipelineRunner(DialogArgs args, IPipelineUseC
         return ui.ShowDialog() == DialogResult.OK ? ui.Result : null;
     }
 
+    public override CohortHoldoutLookupRequest GetCohortHoldoutLookupRequest(ExternalCohortTable externalCohortTable, IProject project, CohortIdentificationConfiguration cic)
+    {
+        // if on wrong Thread
+        if (_mainDockPanel?.InvokeRequired ?? false)
+            return _mainDockPanel.Invoke(() =>
+                GetCohortHoldoutLookupRequest(externalCohortTable, project, cic));
+
+        var ui = new Rdmp.UI.CohortUI.CreateHoldoutLookup.CreateHoldoutLookupUI(this, externalCohortTable,  cic);
+
+        if (!string.IsNullOrWhiteSpace(cic.Description))
+            ui.CohortDescription = $"{cic.Description} ({Environment.UserName} - {DateTime.Now})";
+        return ui.ShowDialog() == DialogResult.OK ? ui.Result : ui.Result;
```

Comment:
I'm thinking this should be ui.Result:null like the method above, instead of returning the result regardless of the user action?